src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb

```diff
@@ -0,0 +1,2111 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9bea0b2f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a directory if it doesn't exist\n",
+    "def make_directory(path):\n",
+    "    if not os.path.exists(path):\n",
+    "        os.mkdir(path)\n",
+    "    \n",
+    "make_directory(\"data\")\n",
+    "make_directory(\"data_pfds\")\n",
+    "make_directory(\"data_tabulated\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "562d5bfd",
+   "metadata": {},
+   "source": [
+    "## Global Solver Settings\n",
+    "\n",
+    "Use the IDAES configuration system for solver settings. These will apply to all Ipopt instances created, including the ones created in initialization methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ac64d3c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_idaes_solver_configuration_defaults()\n",
+    "idaes.cfg.ipopt.options.nlp_scaling_method = \"user-scaling\"\n",
+    "idaes.cfg.ipopt.options.linear_solver = \"ma57\"\n",
+    "idaes.cfg.ipopt.options.OF_ma57_automatic_scaling = \"yes\"\n",
+    "idaes.cfg.ipopt.options.ma57_pivtol = 1e-5\n",
+    "idaes.cfg.ipopt.options.ma57_pivtolmax = 0.1\n",
+    "idaes.cfg.ipopt.options.bound_push = 1e-20\n",
+    "solver = pyo.SolverFactory(\"ipopt\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3aa188b7",
+   "metadata": {},
+   "source": [
+    "## Create the SOFC model\n",
+    "\n",
+    "Create the SOFC model and initialize it or read the saved initialization if available.  The base SOFC model is configured to match the NGFC report."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "2305d41a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_base_sofc_model():\n",
+    "    m = get_model()\n",
+    "    save_name = \"sofc_init.json.gz\"\n",
+    "    if os.path.exists(save_name):\n",
+    "        iutil.from_json(m, fname=save_name, wts=iutil.StoreSpec(suffix=False))\n",
+    "        print(\"Loading model from json\")\n",
+    "    else:\n",
+    "        initialize(m)\n",
+    "        iutil.to_json(m, fname=save_name)\n",
+    "    res = solver.solve(m, tee=True)\n",
+    "    return m"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f37a4d5c",
+   "metadata": {},
+   "source": [
+    "## Design Optimization\n",
+    "Optimize the heat exchanger sizes at the nameplate capacity of 650 MW."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "62aa169f",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2022-11-30 15:17:15 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.prereformer.control_volume.properties_out[0.0]._material_density_term[Vap,CH4]\n",
```

Comment:
Can we address or suppress this output (or clear the output)? With all this output here, it is a lot of scrolling to read the notebook.

Reply:
I think the way to do this is to set the logger level to ERROR.  At least for this one cell.  Then you won't see warnings.